TCD-2622 and TCD-2372: Remove references to SC v4.6.x and add 4.8.0 CLI flags (#1962)
saucelabs/sauce-docs: docs/secure-connections/sauce-connect/security-authentication.md

- **OCSP:** http://gp.symcd.com

> - **OCSP Servers for VDC/RDC clouds:** http://ocsp.digicert.com, http://status.geotrust.com

Reviewer: ```suggestion
```

> - **OCSP Servers for Headless:** http://ocsp.int-x3.letsencrypt.org, http://isrg.trustid.ocsp.identrust.com

Reviewer: ```suggestion
```

> Sauce Connect Proxy reverses tunnel VM-to-test target traffic through the TLS connection from Sauce Connect-to-tunnel endpoints. Your Selenium and Appium webdriver traffic is sent over `http(80)` or `https(443)` to `ondemand.saucelabs.com`, which has its own TLS certificate that's then passed to the test VM.

Reviewer: ```suggestion
Sauce Connect Proxy routes VM-to-test target traffic through the TLS connection between the Sauce Connect Proxy client and the Sauce Connect Server. Sauce Connect Proxy is not used with your Selenium and Appium webdriver traffic to the Sauce Labs `on-demand` endpoint, for example, `ondemand.us-west-1.saucelabs.com`.
```
